solution/ui/regulations/js/src/components/ViewResourcesLink.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <div v-if="isLink()" style="padding-left: 5px; font-size: 12px; margin-top: -10px;">
+      <a v-if="this.count !== '0'" v-on:click="clickHandler">View {{this.section}} resources ({{ this.count }})</a>
+      <div v-else>No resources for {{this.section}}.</div>
```

Comment:
for both the link and the button, the "View § 432.30 resources" part of the button label should be `font-weight: 600`.  The count in parentheses (`(1)`) can say at the default font weight.

Something like:
```
      <button class="btn" v-if="this.count !== '0'" v-on:click="clickHandler">
          <span class="bold">View {{this.section}} resources</span> ({{ this.count }})
      </button>
```